canaries/canary-008-2021-q3.txt

```diff
@@ -0,0 +1,50 @@
+---===[ Grin Canary #008 ]===---
+
+
+Statements
+-----------
+
+The Grin security contacts who have digitally signed this canary state the following:
+
+1. The date of issue of this canary is July 25, 2021.
+
+2. The latest Grin release is v5.1.0 ` 72a6eb8`
+
+3. No warrants have ever been served to us with regard to the Grin
+Project (e.g. to hand out the private signing keys or to introduce
+backdoors).
+
+4. For now, we do not plan to publish another canary due to limited team resources and activity.
+
+Special announcements
+----------------------
+
+This is the last canary to be expected until new active members are added to the security team.
+
+Disclaimers and notes
+----------------------
+
+This canary scheme is not infallible. Although signing the declaration
+makes it very difficult for a third party to produce arbitrary
+declarations, it does not prevent them from using force or other
+means, like blackmail or compromising the signers' laptops, to coerce
+us to produce false declarations.
+
+The block hashes quoted below (Proof of freshness) serve to demonstrate
+that this canary could not have been created prior to the date stated.
+It shows that a series of canaries was not created in advance.
+
+This declaration is merely a best effort and is provided without any
+guarantee or warranty. It is not legally binding in any way to
+anybody. None of the signers should be ever held legally responsible
+for any of the statements made here.
+
+Proof of freshness
+-------------------
+
+$ date -R -u && grin client status | grep 'Last block' | cut -c 18- && curl -s 'https://blockstream.info/api/blocks/tip/hash'; echo && curl -s 'https://api.blockcypher.com/v1/ltc/main' | grep '"hash' | cut -c 12-75 && curl -s 'https://api.blockcypher.com/v1/eth/main' | grep '"hash' | cut -c 12-75
+Sun, 25 Jul 2021 18:09:47 +0000
+0005663a420ed1404c8da37617e78be875a496edd7a61901bb8110536f78e579
+0000000000000000000ce2db6bf4cc41647f3f2af8f7db579d9c6921a5134ddc
```

Comment:
Proof of freshness checks out:
https://blockstream.info/block/0000000000000000000ce2db6bf4cc41647f3f2af8f7db579d9c6921a5134ddc